File: src/test/java/no/nav/pensjon/innsyn/domain/source/HardcodedOmsorg.java
package no.nav.pensjon.innsyn.domain.source;

import no.nav.pensjon.innsyn.domain.Omsorg;
import no.nav.pensjon.innsyn.source.EntityGetter;

import java.util.List;

public class HardcodedOmsorg implements EntityGetter<Omsorg> {

    @Override
    public List<Omsorg> getEntities() {
        return List.of(
                new Omsorg(2018, "kilde1", "type1", "status1"),
                new Omsorg(2019, "kilde2", "type2", "status2"));
    }
}

File: src/test/kotlin/no/nav/pensjon/innsyn/domain/source/HardcodedOmsorg.kt
package no.nav.pensjon.innsyn.domain.source

import no.nav.pensjon.innsyn.domain.Omsorg
import no.nav.pensjon.innsyn.source.EntityGetter

class HardcodedOmsorg : EntityGetter<Omsorg?> {
    override val entities = listOf(
                Omsorg(2018, "kilde1", "type1", "status1"),
                Omsorg(2019, "kilde2", "type2", "status2"))
}